src/sql/workbench/services/notebook/browser/models/notebookModel.ts

```diff
@@ -363,23 +363,41 @@ export class NotebookModel extends Disposable implements INotebookModel {
 						}
 					});
 				}
+				// Get Notebook URI Params & adjust to string
+				let notebookUriParams: string = this.notebookUri.query;
+				// Format query to code cell format
+				notebookUriParams = notebookUriParams.replace(/&/g, '\n').replace(/=/g, ' = ');
+				// Get parameter cell and index to place new notebookUri parameters accordingly
+				let parameterCellIndex = 0;
+				let hasParameterCell = false;
+				let hasInjectedCell = false;
+				let injectedParametersMsg = localize('injectedParametersMsg', '# Injected-Parameters\n');
```

Comment:
Put this as a const in the file so you don't have to evaluate it every time this function is called

Reply:
And that way you can also re-use it instead of redefining it below

Reply:
Will do. Thanks!